client-js/app/elements/labrad-grapher.html

```diff
@@ -123,8 +123,8 @@
       <div id="container">
         <paper-header-panel id="left-column">
           <div class="paper-header" id="buttons">
-            <paper-icon-button icon="stars" id="star" on-click="starClicked" title="show starred items"></paper-icon-button>
-            <paper-icon-button icon="delete" id="trash" on-click="trashClicked" title="show trashed items"></paper-icon-button>
+            <paper-icon-button icon="star" id="star" on-click="starClicked" title="Show only starred items."></paper-icon-button>
+            <paper-icon-button icon="delete" id="trash" on-click="trashClicked" title="Show trashed items."></paper-icon-button>
```

Comment:
No other titles have full stops, but they are usually All Upper Title Case. I don't know if there is a style guide, but that's what I've been using.